MatomoTracker.php

```diff
@@ -1525,6 +1563,7 @@ protected function sendRequest($url, $method = 'GET', $data = null, $force = fal
             // Clear custom variables so they don't get copied over to other users in the bulk request
             $this->clearCustomVariables();
             $this->clearCustomTrackingParameters();
+            $this->clearPerformanceTimings();
```

Comment:
btw... was thinking... should we maybe log a notice or something should it track a request that is not a pageview, and doesn't have a pageview ID set? In this case we wouldn't be able to actually track the data right? Or maybe we would need to not clear the data in that case and wait to see if a trackPageview happens maybe later? Not sure how to handle it best and what is most clear to the user. 

Should maybe also mention this additionally in the doc for the set performance metrics method. Otherwise looks all good

Reply:
@tsteur updated that. The performance timings will now be appended to the first tracking url that includes a pageview (which can be the pageview itself or any later request). The performance timings will then be unset directly. Also added a @deprecated to the generation time method